controllers/gatling_controller.go

```diff
@@ -217,8 +217,9 @@ func (r *GatlingReconciler) gatlingRunnerReconcile(ctx context.Context, req ctrl
 	err := r.Get(ctx, client.ObjectKey{Name: gatling.Status.RunnerJobName, Namespace: req.Namespace}, foundJob)
 	if err != nil && apierr.IsNotFound(err) {
 		duration := utils.GetEpocTime() - gatling.Status.RunnerStartTime
-		if duration > maxJobCreationWaitTimeInSeconds {
-			msg := fmt.Sprintf("Runs out of time (%d sec) in creating the runner job", maxJobCreationWaitTimeInSeconds)
+		maxCreationWaitTIme := int32(utils.GetNumEnv("MAX_JOB_CREATION_WAIT_TIME", maxJobCreationWaitTimeInSeconds))
```

Comment:
I think it is better to name the environment variables the same, Because "InSeconds" gives us the unit of time. 
And why did you exclude "Job" in the "maxCreationWaitTIme" variable name?

```suggestion
		maxJobCreationWaitTime := int32(utils.GetNumEnv("MAX_JOB_CREATION_WAIT_TIME_IN_SECONDS", maxJobCreationWaitTimeInSeconds))
```

Reply:
@banatt 
>I think it is better to name the environment variables the same, because "InSeconds" gives us the unit of time.

For example, in environment variables such as java heap and timeout, I have not seen cases where you add much units. (e.g. HEAP_IN_GB,RDS_TIMEOUT_IN_MSECONDS,START_DATE_IN_UTC)

>And why is 'Job' not included in the 'maxCreationWaitTIme' variable name?

I thought it was a variable name that would not be used all the time and that was sufficient to convey the intent of the variable name.